apps/api-gateway/src/schema/schema.controller.ts

```diff
@@ -43,7 +44,7 @@ export class SchemaController {
   async getSchemaById(
     @Res() res: Response,
     @Param('orgId', new ParseUUIDPipe({exceptionFactory: (): Error => { throw new BadRequestException(ResponseMessages.organisation.error.invalidOrgId); }})) orgId: string,    
-    @Param('schemaId') schemaId: string
+    @Param('schemaId', TrimStringParamPipe) schemaId: string
```

Comment:
Are we allowing creation of schemas with surrounding spaces?

If yes than will this affect it?

Reply:
Resolved